src/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -163,7 +163,7 @@ public static partial class Queryable
         public static System.Linq.IQueryable<TSource> Union<TSource>(this System.Linq.IQueryable<TSource> source1, System.Collections.Generic.IEnumerable<TSource> source2, System.Collections.Generic.IEqualityComparer<TSource> comparer) { throw null; }
         public static System.Linq.IQueryable<TSource> Where<TSource>(this System.Linq.IQueryable<TSource> source, System.Linq.Expressions.Expression<System.Func<TSource, bool>> predicate) { throw null; }
         public static System.Linq.IQueryable<TSource> Where<TSource>(this System.Linq.IQueryable<TSource> source, System.Linq.Expressions.Expression<System.Func<TSource, int, bool>> predicate) { throw null; }
-        public static System.Linq.IQueryable<(TFirst, TSecond)> Zip<TFirst, TSecond>(this System.Linq.IQueryable<TFirst> source1, System.Collections.Generic.IEnumerable<TSecond> source2) { return null; }
+        public static System.Linq.IQueryable<(TFirst first, TSecond second)> Zip<TFirst, TSecond>(this System.Linq.IQueryable<TFirst> source1, System.Collections.Generic.IEnumerable<TSecond> source2) { return null; }
```

Comment:
any reason this uses `return null `when the other methods `throw null`?

Reply:
I probably misread that because `throw null` is terrible (it throws a NullReferenceException before the actual `throw` expression is evaluated) but this is clearly supposed to be garbage for the reference assembly. I'll change to use the existing style.

Note: I think the CLR prevents you from loading reference assemblies as code, so this is probably all moot.

Reply:
We've standardized on `throw null;` as there's nothing else that works consistently and produces as small IL, e.g. `return null;` is similarly just two IL instructions, but it needs to be augmented in methods that have `out` parameters and the like.